Cargo.toml

```diff
@@ -70,6 +71,7 @@ features = ["suggestions", "color", "wrap_help"]
 
 [dev-dependencies]
 pretty_assertions = "1.3.0"
+dirs = "5.0.1"
```

Comment:
I think we already have `dirs` in `[dependencies]` table so there is no need to specify it here.